docs/development/pigs/pig-xxx.rst

```diff
@@ -0,0 +1,182 @@
+* Status: draft
+* Discussion:
+
+Abstract
+========
+
+In this PIG we want to discuss a restructuring of the way light curves are
+computed in Gammapy.
+In general we propose to perform a spectral analysis in each of the time bins
+of the light curve to obtain the integral flux.
+For this purpose the new :class:`LightCurveEstimator` class should essentially
+be a wrapper around the *standard* spectrum analysis of Gammapy using pipeline
+classes (e.g. `gammapy.scripts.SpectrumAnalysisIACT`).
+A further input should of course be the time intervals of the light curve,
+whose computation will be done outside of the new `LightCurve` classes.
+
+It is still under discussion if the LightCurve code should prepare new
+`gammapy.data.ObservationList` s for each light curve bin or modify a
+`~gammapy.data.GTI` table, which the background/spectrum classes would
+respect.
+
+
+Detailed Description
+====================
+
+Background / What we have now
+-----------------------------
+
+The current :class:`gammapy.time.LightCurveEstimator` class takes as input a
+:class:`gammapy.spectrum.SpectrumExtraction` instance for which a list of
+:class:`gammapy.background.BackgroundEstimate` is required.
+Apart from the time intervals, the user also has to provide a
+`gammapy.spectrum.SpectralModel` that is used to compute the expected counts in
+a time bin and to scale the integral flux with respect to the excess events
+found in that time bin.
+The parameters of the spectral model are generally obtained via a spectral fit
+to the whole data sample.
+A tutorial notebook of this approach can be found `here
+<http://docs.gammapy.org/dev/notebooks/light_curve.html>`_.
+
+Drawbacks of this approach are:
+ * the user has to estimate the spectral parameters beforehand, that is before
+   computing a light curve;
+ * changes in the spectral shape between the time bins are not supported.
+
+The current `~gammapy.time.LightCurveEstimator` does not support a 3D cube
+analysis so far.
+
+
+Outline of the new implementation in 1D
+---------------------------------------
+
+In the new approach we want to overcome the two drawbacks mentioned above and
+provide a maximum degree of flexibility to the user when computing the light
+curve.
+
+First, the user should provide a list/table of time intervals for which she/he
+wants to compute the integral flux.
+The computation of this list/table will be done outside of the `LightCurve`
+class, e.g. we could think about extending the `gammapy.data.gti` class to
+provide some utilities for this.
+
+With this list/table the new `LightCurveEstimator` should then prepare/create a
+new `~gammapy.data.ObservatioList` for each time bin, in which the events have
+already been filtered properly and the
+`DataStoreObservation.observation_live_time_duration` were updated accordingly.
+These new `~gammapy.data.ObservationList` can then be passed on to the
+*standard* spectrum analysis with the `gammapy.background` and
+`gammapy.spectrum` modules.
```

Comment:
I am not sure this logic is the best one. We don't really want to recreate a series of `Observation` for a specific time binning, no? This will become very heavy if you perform it on many different scales no?

Isn't it a simpler approach to add an event filter to the spectral extraction? 
After all this is not limited to time at all, one might want to perform a spectral analysis of some event types only. You don't want to recreate an `Observation` for this.

Then if all spectra have their own `TSTART`, `TSTOP` and `ONTIME` there is no difficulty extracting the light curve on any binning scheme build from the input one, by combining spectra during the fit process.